README.md

```diff
@@ -1 +1 @@
-# sentinel
\ No newline at end of file
+# Sentinel
```

Comment:
Maybe we could have a tagline or smth...

Reply:
I agree we should come up with something in our next meeting